_functions.sh

```diff
@@ -106,6 +106,7 @@ Environment Variables
     exogtn         GateIn eXo edition                      - Apache Tomcat bundle
     plf            eXo Platform Standard Edition           - Apache Tomcat bundle
     plfcom         eXo Platform Community Edition          - Apache Tomcat bundle
+    meeds          Meeds.io                                - Apache Tomcat bundle
```

Comment:
IMO, we need to specify which default docker images versions (Onlyoffice, Mysql, Postgres,...) for Meeds edition starting with 1.0 version. See https://github.com/exoplatform/adt/blob/d6c27cdab89d17625f79e0fbf6538c31b0df11a3/_functions.sh#L707
Here a suggestion: 
`      if [[ "${PRODUCT_NAME}" =~ ^(meeds) ]]; then 
           ...........
        if [[ "${PRODUCT_VERSION}" =~ ^(1.0) ]]; then
              env_var "DEPLOYMENT_FORCE_JDBC_DRIVER_ADDON" "true"
              env_var "DEPLOYMENT_ES_IMAGE_VERSION" "1.2.2"
              ...
              configurable_env_var "DEPLOYMENT_ONLYOFFICE_IMAGE_VERSION" "5.4.2.46" 
        else 
              echo_error "Product version \"${PRODUCT_VERSION}\" not yet managed"
              exit 1
        fi`

Reply:
ok, I let you make the modifications onPR @hbenali , thanks !